assessment.rb

```diff
@@ -2,6 +2,8 @@
 # Assessment answers to:
 
 # question 2:
+licensee.create(surname: 'Bishop', given_name: 'Caren', height:63, weight: 132, dob: '1943-09-26', gender: 'f') ? puts('success') : puts('fail')
```

Comment:
Your `create` is mostly correct; however `.create` is a method on the class, Licensee, not a specific `licensee` instance. Additionally, `.create` returns the new instance that it creates, so we probably want to take the output and store it in a variable.

Your logic of using the output as input to a ternary isn't a bad one, but because `.create` returns a new instance rather than `true`/`false`.  `.save`, on the other hand...